documentation/import-from-rdbms.adoc

```diff
@@ -0,0 +1,120 @@
+
+If we don't find any constraints *at all* in your source database, we'll map everyhthing to a node for your convenience. While this saves you doing some mapping manually, please note that a graph with just nodes doesn't make for a connected graph! From here, delete nodes that should be relationships and recreate relationships between the remaining nodes as is most appropriate for your data.
+
+
+[TIP]
+=====
+The generation and mapping doesn't currently support composite primary or foregin keys, so you won't see mappings related to these, nor be able to import them easily via the import UI just yet.
+=====
+
+=== Refining a model
+
+Even in this example where a well-conencted graph model has been generated, the model can benefit from some tidy up. For example, tables mapped to nodes simply use the correspoinding table name as the Label, but we can tidy that up to make it more suitable for our Graph. The same applies for relatonships which by default use a concatenation of related table names. Relationship direction may also benefit from being reversed for the chosen semantics. See the example below where the concepts of Person, Order and Employee and their corresponding relationships are tidied up.
+
+image::import-iterate-graph-model.gif[]
+
+In the Data source panel on the left hand side, blue icons denote that the column form a particular table has been mapped to the graph model. It's worth reviewing the list of tables to any that have not been mapped.
+
+image::import-tables-mapped.png[]
+
+
+If the data is in the right shape, these can be mapped manually to the graph model. You can learn more about how nodes and relationships are mapped in the "Import from CSV" guide that covers the same concepts required here. It explains node IDs and relationship mappings in more details with CSV files instead of tables.
+
+When all parts of the graph model (nodes and relationships) are successfully mapped to tables, they will be annotated with a green check mark. If an import is attempted and anything is missing, it will be higlighted in red to be fixed before proceeding with the import. 
+
+== Running an import job
+
+Once you're happy with your graph model and how your tables are mapped to it, you can submit the import job. If you're already connected to an instance, this instance will be used, otherwise you'll be prompted to select an instance.
```

Comment:
```suggestion
Once you're happy with your graph model and how your tables are mapped to it, you can runthe import job.
If you're already connected to an instance, this instance is used. 
Otherwise, you are prompted to select an instance.
```